plottable.d.ts

```diff
@@ -277,6 +277,19 @@ declare module Plottable {
 
 
 declare module Plottable {
+    module Utils {
+        class CallbackSet<CB extends Function> {
+            constructor();
+            add(value: CB): CallbackSet<CB>;
+            remove(value: CB): CallbackSet<CB>;
```

Comment:
Note that a bunch of Java API and I think a bunch of other API have add/remove actually return a boolean.  I will note that because of our style, I think it is correct in this case that we return `this` and I actually personally disagree with the conventional API.  I am mainly noting that I think this is outside convention.  Not incorrect, but a tiny bit unconventional.